endpoints-support.md

```diff
@@ -220,115 +220,116 @@
-| <sub>/rest/storage-volumes/{id}</sub>                                                           |            PUT            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    | :white_check_mark:       | :white_check_mark:       | :white_check_mark:       | :white_check_mark: |:white_check_mark:| :white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
-| <sub>/rest/storage-volumes/{id}</sub>                                                           |          DELETE           |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    | :white_check_mark:       | :white_check_mark:       | :white_check_mark:       | :white_check_mark: |:white_check_mark:| :white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
+| <sub>/rest/storage-volumes</sub>                                                                |            GET            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |
+| <sub>/rest/storage-volumes/{id}</sub>                                                           |            GET            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |
+| <sub>/rest/storage-volumes</sub>                                                                |           POST            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |
+| <sub>/rest/storage-volumes/{id}</sub>                                                           |            PUT            |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |
+| <sub>/rest/storage-volumes/{id}</sub>                                                           |          DELETE           |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |    :white_check_mark:    |
+s
```

Comment:
There is an extra row added at the end with s. Please check it.

Reply:
Fixed the change